mobile/src/main/java/org/openhab/habdroid/ui/WidgetListFragment.kt

```diff
@@ -113,11 +109,11 @@ class WidgetListFragment : Fragment(), WidgetAdapter.ItemClickListener {
         AlertDialog.Builder(context)
                 .setTitle(R.string.nfc_dialog_title)
                 .setItems(labels.toTypedArray()) { _, which ->
-                    if (which == commands.size + 1 && widget.linkedPage != null) {
-                        createShortcut(context, widget.linkedPage)
+                    if (which == commands.size + 1 && item.linkedPage != null) {
+                        createShortcut(context, item.linkedPage)
                     } else {
-                        val itemToHandle = if (which < commands.size) widget.item else null
-                        val linkToHandle = if (which == commands.size) widget.linkedPage?.link else null
+                        val itemToHandle = if (which < commands.size) item.item else null
```

Comment:
IMO `item` should be renamed to `widget`. I know that two meanings of item conflict here (openHAB Item vs. thing that was clicked), but `item.item` seems too strange to me.

Reply:
I agree, but there's only suboptimal solutions here:
- suboptimal naming (what we have now)
- name the parameter `widget`, which yields a warning as the parameter is named `item` in the base class
- name the parameter `item` and create a variable `widget` from it, which yields a warning because the variable is redundant

:-/

Reply:
I like option 2 and suppressing the warning most.

Reply:
I totally forgot that listener is our code as well ... that makes fixing this pretty easy actually: https://github.com/openhab/openhab-android/pull/1356/commits/b30e0912b1a2f48a905debffb0f4391f4ae0b088